### ISSUE TITLE
Async select options

### ISSUE THREAD

**Issue by tb** [mseemann/angular2-mdl-ext#410]
When set people https://github.com/mseemann/angular2-mdl-ext/blob/master/src/e2e-app/app/select/select.component.ts#L23 async, i.e.
```
  people: any[] = [];
  
  ngOnInit() {
    // ...
    setTimeout(() => {
      this.people = null;
      this.people = [
        {id: 1, name: 'Bryan Cranston'},
        {id: 2, name: 'Aaron Paul'},
        {id: 3, name: 'Bob Odenkirk'},
        {id: 4, name: 'Dean Norris'},
        {id: 5, name: 'Jonathan Banks'},
        {id: 6, name: 'Seal Henry Olusegun Olumide Adeola Samuel'},
        {id: 7, name: 'Rami Malek'},
        {id: 8, name: 'Christian Slater'},
        {id: 9, name: 'Martin Wallström'},
        {id: 10, name: 'Angela Sarafyan'},
        {id: 11, name: 'Robert Pattinson'},
        {id: 12, name: 'Billy Bob Thornton'}
      ];
    }, 250);
  }
```
then initial selection of is not rendered.

Dont know how to fix it yet.

**Comment by mseemann** [comment]
right. there are some problems. the first one is that we use `changeDetection: ChangeDetectionStrategy.OnPush` in the select example. OnPush is still an issue if one uses `ControlValueAccessor` (https://github.com/angular/angular/issues/10816). Thats why `optionComponents: QueryList<MdlOptionComponent>` did not fire on changes. The next problem is:

```
    public ngAfterViewInit() {
        this.bindOptions();
        this.renderValue(this.ngModel);
        this.optionComponents.changes.subscribe(() => this.bindOptions());
    }
```

I think  `this.renderValue(this.ngModel);` should be part of the `bindOptions` function

**Comment by mseemann** [comment]
published as select@0.9.1
